src/main/js/core/Shape/shapeDefinitions.js

```diff
@@ -327,4 +327,36 @@ export const RIGHT_CHEVRON = {
     path: { d: "M37.7 24L14.2 48l-3.9-3.8L30 24 10.3 3.8 14.2 0z" },
     options: { x: -6, y: -6, scale: 0.25 }
 };
+
+/**
+ * The S_0013_line.svg SVG file as an object.
+ *
+ * @private
+ * @type {{path: {d: string}, options: {x: number, y: number, scale: number}}}
+ */
+export const LINE = {
+    path: { d: "M15 0h48v5H10V0" },
+    options: { x: -1, y: -1, scale: 0.25 }
+};
+
+/**
+ * The S_0014_dashed_line.svg SVG file as an object.
+ *
+ * @private
```

Comment:
nit: private keyword can be removed

Reply:
[Fixed](https://github.com/cerner/carbon-graphs/pull/297/files#diff-09b94496871691193f1dd1f077f89851R343)